package.json

```diff
@@ -38,6 +38,7 @@
         "@babel/preset-react": "7.16.7",
         "@babel/register": "^7.16.0",
         "@babel/runtime-corejs3": "7.16.8",
+        "@exabyte-io/made.js": "2022.6.15-0",
```

Comment:
I don't think we need this change

Reply:
hmm but then I need to add it to `peerDependencies`, otherwise ESLint throws an `import/no-extraneous-dependencies` error